### DISCUSSION TITLE
GitHub Actions is currently Disabled for u r acc

### DISCUSSION THREAD

**Discussion by atmosgdrive** [community/community#33100] [Actions]
i test my codes using action for deployment and also for some bug fix purpose. But by 15/sep/2022 morning it said GitHub Actions is currently Disabled for your Account. I request the reason for this action. I wanna know if there is any mistake from my side (or) it's A bug.

- Thank You

**Comment by airtower-luna** [comment]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

What I can say from reading on the forum is that your workflow must've tripped some abuse detection algorithm, one relatively common reason seems to be running an RDP server in a workflow. :sweat_smile:
